src/components/LeaderboardRankingsCard.vue

```diff
@@ -50,20 +53,35 @@ export default {
         return u
       })
     },
+    sortedUsers: function () {
+      return this.leaderboard.users.slice().sort((a, b) => b.points - a.points)
+    },
+    isCurrentUserRanked() {
+      return this.rankedUsers.some(user => user.userId === this.currentUser.id && user.rank <= this.lastRank)
+    },
+    lastRank() {
+      if (this.rankedUsers.length === 0) return 0
+
+      if (typeof this.leaderboard.rankingsTopN === 'number') {
```

Comment:
looks okay if we jst check `this.leaderboard.rankingsTopN`. If its null, hits the else

Reply:
👍